ecommerce_api_client/tests/test_auth.py

```diff
@@ -1,38 +1,68 @@
 # -*- coding: UTF-8 -*-
+
+import datetime
 from unittest import TestCase
 
 import httpretty
 import jwt
+import mock
 import requests
 
-from ecommerce_api_client.auth import JwtAuth, BearerAuth
+from ecommerce_api_client import auth
+from ecommerce_api_client.auth import BearerAuth
```

Comment:
If you're importing `auth`, you can access `BearerAuth` and `JwtAuth` via `auth.BearerAuth` and `auth.JwtAuth`. There is no need to import them again.